pkgs/tools/text/mmdoc/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmark-gfm
+, xxd
+, fastJson
+, libzip
+, ninja
+, meson
+, pkg-config
+}:
+
+let
+
+cmark-gfm' = cmark-gfm.overrideAttrs (oldAttrs: rec {
+  # remove when https://github.com/github/cmark-gfm/pull/248 merged and released
```

Comment:
Why not apply this in the main `cmark-gfm` package?

Reply:
Sure, done!